test-data/unit/check-incremental.test

```diff
@@ -4358,6 +4358,22 @@ import b
 [file b.py.2]
 # uh
 -- b gets rechecked because it changed, but nothing is stale
--- since the interface didn't change
+-- since the interface did not change
 [stale]
 [rechecked b]
+
+[case testParentPatchingMess]
+# cmd: mypy -m d d.k d.k.a d.k.v t
+[file d/__init__.py]
+[file d/k/__init__.py]
+from d.k.a import x
+[file d/k/a.py]
+x = 10
+[file d/k/v.py]
+from d.k.e import x  # type: ignore
```

Comment:
Can you probably add `# flags: --follow-imports=skip` instead of type ignore here to perfectly match the real world crash?